packages/gateway/src/graphql/config.ts

```diff
@@ -104,26 +103,28 @@ export const getApolloConfig = (): Config => {
     introspection: true,
     context: async ({ request, h }) => {
       try {
-        const userId = getUserId({
-          Authorization: request.headers.authorization
-        })
-        let user: IUserModelData | ISystemModelData
-        user = await getUser(
-          { userId },
-          { Authorization: request.headers.authorization }
+        const tokenPayload = getTokenPayload(
+          request.headers.authorization.split(' ')[1]
         )
-        if (!user) {
+        const userId = tokenPayload.sub
+        let user: IUserModelData | ISystemModelData
+        const isSystemUser = tokenPayload.scope.indexOf('recordsearch') > -1
+        if (isSystemUser) {
```

Comment:
@euanmillar event notifications will also go through gateway so we need check for that scope as well

Reply:
@Zangetsu101 I think that they are fine as they do not go through Apollo

Reply:
Okay yeah you're right